### ISSUE TITLE
Strange name suggested when duplicating a file

### ISSUE THREAD

**Issue by novoselt** [sagemathinc/cocalc#7984]
![image](https://github.com/user-attachments/assets/b3f39fe2-e077-470a-8c3f-c0a96df57b01)

I am not sure what's the algorithm is (trying to guess the numbering convention dropping off fractional part and inserting leading zeros to match the old field width???), but I think there should be none at all - the starting point for the duplicate name is exactly the same as the old one and I will tweak it however I want.

**Comment by williamstein** [comment]
> I think there should be none at all - the starting point for the duplicate name is exactly the same as the old one and I will tweak it however I want.

I disagree, though we should add a button to reset the filename to be the same as the original file, which you could then click.

You're right about the algorithm.  It also takes into account other files in the same directory to avoid overwriting an existing file.
